app/serializers/current_user_serializer.rb

```diff
@@ -141,6 +142,15 @@ def include_user_admin_sidebar?
     object.admin?
   end
 
+  def show_experimental_flags_admin_page
+    object.admin? &&
+      object.in_any_groups?(SiteSetting.experimental_flags_admin_page_enabled_groups_map)
```

Comment:
```suggestion
    object.in_any_groups?(SiteSetting.experimental_flags_admin_page_enabled_groups_map)
```

Reply:
The `object.admin?` check has already been done in `include_show_experimental_flags_admin_page?`

Reply:
Oh good point, never thought of it this way -- use_admin_sidebar has the same problem. Will change